velox/functions/prestosql/MapFromEntries.cpp

```diff
@@ -130,7 +132,11 @@ class MapFromEntriesFunction : public exec::VectorFunction {
         const auto size = inputArray->sizeAt(row);
         const auto offset = inputArray->offsetAt(row);
 
-        for (auto i = 0; i < size; ++i) {
+        // We need this loop separated from the next one, because errors in the
```

Comment:
I wonder why don't we need to set mutableSizes[row] = 0 when we encounter a problem with the keys?

Reply:
that would work also i think